### DISCUSSION TITLE
[Suggestion] Change textboxes for specifying paths to file explorer buttons

### DISCUSSION THREAD

**Discussion by PladsElsker** [ModelSurge/sd-webui-comfyui#135] [General]
This would be in the settings for ComfyUI's location, and in the tab if ComfyUI is not found.

**Comment by ljleb** [comment]
Not sure how practical this is for settings, but it makes sense for the install tab.

I have yet to come across an extension that bothers to add a button in the settings, people seem to be fine with how things are.
